pmd-visualforce/src/test/java/net/sourceforge/pmd/lang/vf/VFTestContstants.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.vf;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.List;
+
+public abstract class VFTestContstants {
+    private static final Path ROOT_PATH = Paths.get("src", "test", "resources", "net", "sourceforge",
```

Comment:
I'd like to have the test resources in a more specific directory. If you just look at the directory listing now (e.g. "src/test/resources/net/sourceforge/pmd/lang/vf/metadata-sfdx/pages/StandardAccount.page") you have no clue, what this is for (in which unit test or rule test this is used, etc.). Maybe we keep adding files to these directories for multiple different rules (because there are already pages, so we add another page in the same directory, but test something completely independent)... IMHO we should avoid this. I'd prefer a own directory at least per test class (and the package/directory name gives a hint where the test class is and its name).